src/app/taskBoard/TaskBoard.tsx

```diff
@@ -104,25 +109,45 @@ export const TaskBoard = () => {
     [lists, moveCard],
   )
 
+  const { showModal } = useSelector(selectCreateTask)
+
   return (
-    <Stack direction="row" columnGap={2} sx={{ padding: '0px 20px', width: '100vw' }}>
-      {Object.entries(lists).map(([listId, listData], index) => (
-        <Droppable
-          key={listId}
-          accept={'taskCard'}
-          index={index}
-          id={Number(listId)}
-          onDropItem={(item) => onDropItem(Number(listId), item)}
+    <AppMargin size={SizeofAppMargin.LARGE} ptb="18.5px">
```

Comment:
@aatbip let's use `py` instead f `ptb`

Reply:
uses `py` in 8e15b9b32c30e491cb012f96721c4dfbfa8790cb commit